### PR TITLE
docs: update contributing.md and package-generator

### LINKED ISSUE #815: Revive package naming convention in contributing
There was a separate guide dedicated to entities naming in reatom. Package naming should be restored and put in contributing.md st least.

### PR DESCRIPTION
**Pull request by ghost**
Closes #815

### CODE REVIEW

File: docs/src/content/docs/contributing.md [hunk 1]
```diff
@@ -3,88 +3,90 @@ title: Contributing
...
    ```
-   Replace `PACKAGE_NAME` with the name of the package you want to build. For example, `npm-react` or `persist`. The [turbo](https://turbo.build) will handle all dependencies and build all required graph.
-4. 🎨 Create a `feature-branch` from `v3` branch that starts from the number of the [created issue](#creating-an-issue).
-5. 🧪 If you want to fix a bug, write reproduction tests first.
-6. 🪄 Make changes.
-7. 🧪 If you added a feature, write tests for it.
-8. 📝 Record the changes according to [conventional rules](#commit-rules).
-   ```bash
-   $ git commit -m "<type>[optional scope]: <description>"
+   Dependencies are installed for all packages, but only packages included in `@reatom/framework` (`core`, `utils`, `async` etc.) are built.
```
Comment (BANOnotIT) [inline docs/src/content/docs/contributing.md]: I don't think that declarative sentence works here.
```suggestion
   Note: this command installs dependencies for all packages, but only builds packages included in `@reatom/framework`, like `core`, `utils`, `async`, etc.
```

File: docs/src/content/docs/contributing.md [hunk 2]
```diff
@@ -3,88 +3,90 @@ title: Contributing
...
-Edit the `author` field and add yourself to the `maintainers` list of the `package.json` of the new package.
+Add needed dependencies by running `npm install` in your package's directory. If you're making an adapter for a particular npm library (like `@reatom/npm-react` for React), the library should be saved as peer: `npm install --save-peer <LIBRARY>`
+
+<!-- ??? -->
+<!-- To add dependencies, add them manually to the `package.json` of the new package and install them from the root of the repo. -->
+
+### Package naming rule
 
-If you need to add dependencies, add them manually to the `package.json` of the new package and install them from the root of the repo.
+Packages that integrate Reatom with external APIs should have their names prefixed with one of the following strings: `node-`, `npm-`, `web-`. For example, [`@reatom/npm-history`](https://reatom.dev/package/npm-history) provides an adapter for the [`history`](https://npmjs.com/history) package. Similiarly, a potential adapter for [Web History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) would be named `@reatom/web-history`.
```
Comment (BANOnotIT) [inline docs/src/content/docs/contributing.md]:
```suggestion
Packages that integrate Reatom with external APIs should have their names prefixed with API or library source kind: `node-`, `npm-`, `web-`. For example, [`@reatom/npm-history`](https://reatom.dev/package/npm-history) provides an adapter for the [`history`](https://npmjs.com/history) package. Similarly, a potential adapter for [Web History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) would be named `@reatom/web-history`.
```

File: docs/src/content/docs/contributing.md [hunk 3]
```diff
@@ -3,88 +3,90 @@ title: Contributing
...
+- `docs`: documentation only changes
+- `ci`: a change made to CI configurations and scripts
+- `style`: cosmetic code change
+- `test`: change that only adds or corrects tests
+- `revert`: change that reverts previous commits
 
 ### Allowed `<scope>`
 
-Directory name from `/packages/<scope>`
+Name of directory of a package (`/packages/<scope>`)
```
Comment (BANOnotIT) [inline docs/src/content/docs/contributing.md]:
```suggestion
Package directory name. Eg: `/packages/effects` is scoped as `effects` 
```

File: docs/src/content/docs/contributing.md [hunk 4]
```diff
@@ -3,88 +3,90 @@ title: Contributing
...
 
-### Style for `<description>`
+### `<description>` rules
 
-- only English language
-- Use imperative, present tense: `change` not `changed` nor `changes`
-- don't capitalize first letter
-- no dot (`.`) at the end
+- should be written in English
+- should be in imperative form (like `change` instead `changed` or `changes`)
```
Comment (BANOnotIT) [inline docs/src/content/docs/contributing.md]:
```suggestion
- should be in imperative tense (like `change` instead `changed` or `changes`)
```

File: docs/src/content/docs/contributing.md [hunk 5]
```diff
@@ -3,88 +3,90 @@ title: Contributing
...
+- should not be capitalized
+- should not have period at the end
```
Comment (BANOnotIT) [inline docs/src/content/docs/contributing.md]:
```suggestion
- should not have period (`.`) at the end
```

**Review by BANOnotIT — CHANGES_REQUESTED** [review]

File: docs/src/content/docs/contributing.md (hunk 4 above)
Comment (ghost) [reply docs/src/content/docs/contributing.md]: [Imperative is a mood, not a tense](https://en.wikipedia.org/wiki/Imperative_mood)

File: docs/src/content/docs/contributing.md [hunk 6]
```diff
@@ -45,7 +45,7 @@ Add needed dependencies by running `npm install` in your package's directory. If
 
 ### Package naming rule
 
-Packages that integrate Reatom with external APIs should have their names prefixed with one of the following strings: `node-`, `npm-`, `web-`. For example, [`@reatom/npm-history`](https://reatom.dev/package/npm-history) provides an adapter for the [`history`](https://npmjs.com/history) package. Similiarly, a potential adapter for [Web History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) would be named `@reatom/web-history`.
+Packages that integrate Reatom with external APIs should have their names prefixed with API or library source kind: `node-`, `npm-`, `web-`. For example, [`@reatom/npm-history`](https://reatom.dev/package/npm-history) provides an adapter for the [`history`](https://npmjs.com/history) package. Similarly, a potential adapter for [Web History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) would be named `@reatom/web-history`.Name of directory of a package (`/packages/<scope>`)
```
Comment (BANOnotIT) [inline docs/src/content/docs/contributing.md]:
```suggestion
Packages that integrate Reatom with external APIs should have their names prefixed with API or library source kind: `node-`, `npm-`, `web-`. For example, [`@reatom/npm-history`](https://reatom.dev/package/npm-history) provides an adapter for the [`history`](https://npmjs.com/history) package. Similarly, a potential adapter for [Web History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API) would be named `@reatom/web-history`.
```

**Review by BANOnotIT — DISMISSED** [review]

**Review by artalar — APPROVED** [review]

**Pull request merged**